prometheus/src/index.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  DefaultDetailsViewSection,
+  DetailsViewSectionProps,
+  registerDetailsViewSection,
+  registerDetailsViewSectionsProcessor,
+} from '@kinvolk/headlamp-plugin/lib';
+import Alert from '@material-ui/lab/Alert';
+import React, { useEffect, useState } from 'react';
+import { GenericMetricsChart } from './common';
+import { isPrometheusInstalled } from './request';
+
+function MetricsEnabled({ children }) {
+  const [prometheusInstalled, setPrometheusInstalled] = useState<boolean>(false);
+  const [promPodName, setPromPodName] = useState<string>(null);
+  const [promNamespace, setPromNamespace] = useState<string>(null);
```

Comment:
I think a modern version of React does batch the state changes, so this should be okay. But this these are all related, maybe we could have it as: `useState<{podName: string; podNamespace: string} | null>`, null being "Not installed".